.changeset/unlucky-cats-lie.md

```diff
@@ -0,0 +1,6 @@
+---
+'graphql-yoga': minor
+---
+
+Customize the landing page by passing a custom renderer that returns `Response` to the `landingPage`
```

Comment:
we should add a simple example of usage in the changeset

Reply:
+ also documentation

Reply:
Done!
https://github.com/dotansimha/graphql-yoga/pull/3332/commits/4c292f8b3bef6c681dfd01f20a07b8eba88671aa
Let me know what you think.

Reply:
Also added an example to the changeset.
https://github.com/dotansimha/graphql-yoga/pull/3332/commits/f337d7bf2ab24af33a73d1e73320ae9b272eaeae
Let me know what you think.